lxdock/client.py

```diff
@@ -1,6 +1,24 @@
+#!/usr/bin/python3
+import ast
+import asyncio
 import pylxd
+import pyslp
+import random
 
+from pyslp.slptool import SLPClient
 
-def get_client():
+
+def get_slp_hosts():
+    loop = asyncio.get_event_loop()
+    slp_client = SLPClient(ip_addrs=['0.0.0.0'])
+    url_entries = loop.run_until_complete(
+        slp_client.findsrvs(service_type="lxd_host")
+    )
+    host_list = url_entries
+    return host_list
+
+def get_client(host_list=get_slp_hosts()):
```

Comment:
Python only evaluates default values once when the function is loaded. It might be better to default this to `None` and then fallback to `get_slp_hosts()`.

e.g.
```python
def get_client(hosts=None):
    hosts = hosts or get_slp_list()
    ...
```